src/librustc/hir/map/mod.rs

```diff
@@ -626,6 +632,34 @@ impl<'hir> Map<'hir> {
         }
     }
 
+    pub fn get_return_block(&self, id: NodeId) -> Option<NodeId> {
```

Comment:
Comment explaining what this function does would be good. Ideally with an example.